elements/pfe-sass/maps/_typography.scss

```diff
@@ -81,3 +81,148 @@ $pf-type-sizing--titles: (
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm),
+    ),
+    lg: (
+      FontFamily: map-get($pfe-typography-base, font-family--heading),
+      FontSize:   map-get($pf-type-sizing--titles, title--m-lg--FontSize),
+      LineHeight: map-get($pfe-typography-base, line-height),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm),
+    ),
+    md: (
+      FontFamily: map-get($pfe-typography-base, font-family--heading),
+      FontSize:   map-get($pf-type-sizing--titles, title--m-md--FontSize),
+      LineHeight: map-get($pfe-typography-base, line-height),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm),
+    ),
+    sm: (
+      FontFamily: map-get($pfe-typography-base, font-family--heading),
+      FontSize:   map-get($pf-type-sizing--titles, title--m-sm--FontSize),
+      LineHeight: map-get($pfe-typography-base, line-height),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm),
+    ),
+    xs: (
+      FontFamily: map-get($pfe-typography-base, font-family--heading),
+      FontSize:   map-get($pf-type-sizing--titles, title--m-xs--FontSize),
+      LineHeight: map-get($pfe-typography-base, line-height),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm),
+    )
+  ),
+  text: (
+    xl: (
+      FontFamily: map-get($pfe-typography-base, font-family),
+      FontSize:   map-get($pf-type-sizing, FontSize--xl),
+      LineHeight: map-get($pfe-typography-base, line-height),
+      FontWeight: map-get($pfe-typography-base, font-weight--light),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--md),
+    ),
+    // @TODO Update this to match above; this is a temp test for the pfe-text mixin
+    md: (
+      FontSize: pfe-var(content--FontSize)
+    ),
+    sm: (
+      FontSize: pfe-var(content--small--FontSize)
+    )
+  ),
+  /// The variable stack includes a hook just for headings when they appear in the 
+  /// light DOM within components: --pf-c--FontSize
+  ///
+  /// Components can't control the font size of block level elements like HTML headings 
+  /// when they are in the light DOM.  A component _could_ redefine the variables for 
+  /// standard headings by using the variable hooks such as: `--pf-c--content--h4--FontSize` 
+  /// but it can't know if the light DOM might contain an H1, H2, H3, etc. it shouldn't
+  /// redefine the variables for all possible headings. Instead, there is one dedicated
+  /// variable for this purpose. 
+  content: (
+    h1: (
+      FontFamily: map-get($pfe-typography-base, font-family),
+      FontSize:   var(--pf-c--FontSize, pfe-var(content--h1--FontSize)),
```

Comment:
thinking aloud: originally I think I wanted the `--pf-c--FontSize` var to only appear sometimes in the stack... but if its always empty by default, maybe this is beneficial to have it *always* in the stack. The only potential downside is developers not knowing when to leverage it...